src/seedu/addressbook/AddressBook.java

```diff
@@ -257,16 +264,17 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
-            showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
+        if (args.length > ARGUMENTS_ONE) {
+        	String[] arrStr = {MESSAGE_INVALID_PROGRAM_ARGS};
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)